### DISCUSSION TITLE
MacArthur HAT for RPi4 is now available

### DISCUSSION THREAD

**Discussion by EarlWer** [bareboat-necessities/lysmarine_gen#298] [General]
Are you planning to get and document the installation of the MacArthur HAT and Power Supply with BBN?

I have one and I will be trying both BBN and OpenMarine to check compatibility.
I am assuming that the 2 RPi systems are very similar.
It should make connecting my new Raymarine ST-2000 auto-tiller a lot simpler...

**Comment by mgrouch** [comment]
Sure. Without hardware though I’ll be making just educated guesses. If you are trying with BBN OS and send me steps I’ll gladly include them in the docs. 

Thanks
